mkdocs.yml

```diff
@@ -44,7 +44,7 @@ theme:
 extra:
   social:
     - icon: fontawesome/brands/github
-      link: https://github.com/pyapp-kit/magicgui
+      link: https://github.com/kreshuklab/plant-seg
```

Comment:
Thanks lol

Reply:
Thanks for looking into it! But it's still on-going! I will ask for a review when ready :)